frontend/src/container/Tracim.jsx

```diff
@@ -488,14 +488,24 @@ export class Tracim extends React.Component {
                   )}
                 />
 
-                <Route
-                  path={PAGE.WORKSPACE.PUBLICATION(':idws')}
-                  render={() => (
-                    <div className='tracim__content fullWidthFullHeight'>
-                      <Publications />
-                    </div>
-                  )}
-                />
+                <Switch>
+                  <Route
+                    path={PAGE.WORKSPACE.PUBLICATION(':idws', ':idcts')}
+                    render={() => (
+                      <div className='tracim__content fullWidthFullHeight'>
+                        <Publications />
+                      </div>
+                    )}
+                  />
+                  <Route
+                    path={PAGE.WORKSPACE.PUBLICATIONS(':idws')}
+                    render={() => (
+                      <div className='tracim__content fullWidthFullHeight'>
+                        <Publications />
+                      </div>
+                    )}
+                  />
```

Comment:
the path props accepts an array of strings, I think it could factorize this and you wouldn't require the switch